pkg/zsio/table/writer.go

```diff
@@ -19,7 +21,12 @@ type Table struct {
 
 func NewWriter(w io.Writer) *Table {
 	writer := tabwriter.NewWriter(w, 0, 8, 1, ' ', 0)
-	return &Table{Writer: w, table: writer, limit: 1000}
+	return &Table{
```

Comment:
doesn't this also need a call to Flatten() in Write()?

Reply:
oopsies.  yes